cypress/e2e/functional/opal/manualAccountCreation/accountCreationPage/PO-346_Business_Unit_And_Defendant_type.feature

```diff
@@ -142,7 +142,7 @@ Feature: PO-346 business unit and defendant type Users associated to one and onl
     When I select the "Fine" radio button
     And I see the heading under the fine radio button is "Defendant type"
     And I see "If sole trader, choose 'Adult or youth only'" below the defendant type subheading
-    And I click on cancel
+    And I click on "Cancel"
```

Comment:
This needs updating to use the existing step

Reply:
updated and pushed code